Changelog.md

```diff
@@ -1,6 +1,21 @@
 ## Ignition Rendering
 
-### Ignition Rendering 6.X
+### Ignition Rendering 6.5.0 (2022-06-15)
+
+1. Bounding Box Camera
+    * [Pull request #334](https://github.com/ignitionrobotics/ign-rendering/pull/334)
```

Comment:
Should these links be updated to `gazebosim/gz-rendering` instead of `ignitionrobotics/ign-rendering`?
```suggestion
    * [Pull request #334](https://github.com/gazebosim/gz-rendering/pull/334)
```